Hello_Serialization_stud/Hello_Serialization_stud/Program.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization.Formatters.Binary;
+using System.Runtime.Serialization.Formatters.Soap;
+using System.Xml.Serialization;
+using System.Runtime.Serialization;
+
+namespace Hello_Serialization_stud
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var student1 = new Student();
+            student1.FirstName = "Marco";
+            student1.LastName = "Polo";
+            student1.Nationality = "Venice";
+            student1.SetAddress("Hreschatyk", "10");
+            Console.WriteLine(student1);
+
+            var student2 = new Student();
+            student2.FirstName = "Cristo";
+            student2.LastName = "Columb";
+            student2.Nationality = "Spanish";
+            student2.SetAddress("Shevchenka", "20");
+            Console.WriteLine(student2);
+
+            var students = new List<Student>();
+            students.Add(student1);
+            students.Add(student2);
+
+            BinarySerialize(students);
+
+            SoapSerialize(students);
+
+            XmlSerialize(student1);
+            XmlSerialize(student2);
+        }
+
+        public static void BinarySerialize(List<Student> students)
+        {
+            var binaryFilePath = "binary.bin";
+            Console.WriteLine("BinarySerialize ......");
+
+            using (Stream streamWrite = new FileStream(binaryFilePath, FileMode.Create, FileAccess.Write, FileShare.None))
+            {
+                IFormatter serializeFormatter = new BinaryFormatter();
+
+                foreach (var student in students)
+                {
+                    serializeFormatter.Serialize(streamWrite, student);
+                }
+
+                streamWrite.Close();
+            }
+            
+            using (Stream streamRead = new FileStream(binaryFilePath, FileMode.Open, FileAccess.Read, FileShare.Read))
+            {
+                IFormatter deserializeFormatter = new BinaryFormatter();
+
+                var deserializedStudent = (Student)default;
+
+                do
```

Comment:
what the point to have this do? you expect that in some moment this file could be changed if there will be problem?

Reply:
In Binary binary serialize example I used list of two objects student.
![image](https://user-images.githubusercontent.com/35422255/170566412-393dae39-58b5-442d-9bce-877e9e9ea40f.png)

So, I need loop to read more than one object.